test/fixtures/api/graphql/commit-queries.ts

```diff
@@ -28,12 +29,11 @@ const query = `query ($owner: String!, $repo: String!, $per_page: Int!, $cursor:
     }
   }`;
 
-export const commitsNoLastCursor = {
+
+export const commitsNoLastCursor = (variables) => ({
 	query,
-	variables: {
```

Comment:
moved these values out of here and into the test file, was easier to understand test with this data available.....